apps/server/src/graphql/Task.ts

```diff
@@ -375,6 +392,52 @@ When the task is:
   })
 );
 
+builder.mutationField("makeTaskSubtask", (t) =>
+  t.prismaFieldWithInput({
+    type: "Task",
+    description: "Make a task a subtask of another task. Takes in the ID of the parent task and the ID of the subtask",
+    input: {
+      parentTaskId: t.input.id({ required: true, description: "The ID of the parent Task" }),
+      subTaskId: t.input.id({ required: true, description: "The ID of the subtask" }),
+    },
+    resolve: async (query, _, args) => {
+      const { parentTaskId, subTaskId } = args.input;
+
+      // Fetch the parent task and the subtask from the database
+      const parentTask = await prisma.task.findUnique({
+        where: { id: parseInt(parentTaskId) },
+      });
+      const subTask = await prisma.task.findUnique({
+        where: { id: parseInt(subTaskId) },
+      });
+
+      // Make sure both tasks exist
+      if (!parentTask) {
+        throw new Error(`Parent task with ID ${parentTaskId} does not exist.`);
```

Comment:
GraphQL Yoga (the GraphQL server used in this app) [masks errors of instance](https://the-guild.dev/graphql/yoga-server/tutorial/basic/09-error-handling) `Error`. You need to throw it as a `GraphQLError` instead.
```suggestion
        throw new GraphQLError(`Parent task with ID ${parentTaskId} does not exist.`);
```